common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -744,7 +747,12 @@ def create_response_element(self, **kwargs):
                 hintgroup_element.set("hintfn", hint_fn)
 
         for additional_answer in additional_answers:
-            etree.SubElement(response_element, "additional_answer").text = additional_answer
+            additional_node = etree.SubElement(response_element, "additional_answer")  # pylint: disable=no-member
```

Comment:
I think it's just that our python is one step too sophisticate for what pylint can figure out. Notice the rest of that file uses that construct in like a zillion place. I just copied from the rest of the file to fit how it works.

Someday perhaps I'll form up the argument that the data-xml for tests should just be in ..... xml! My gut is that you want to KISS even more than usual with your tests, but I haven't written a manifesto or anything about it yet. Anyway, in this case, were just sort of "when in Rome" ing it.